main/src/ca/uwaterloo/flix/language/phase/unification/BddFormula.scala

```diff
@@ -15,10 +15,195 @@
  */
 package ca.uwaterloo.flix.language.phase.unification
 
+import ca.uwaterloo.flix.language.ast.{SourceLocation, Symbol, Type}
+import ca.uwaterloo.flix.util.InternalCompilerException
+import ca.uwaterloo.flix.util.collection.Bimap
+import com.github.javabdd.{BDD, BDDFactory, JFactory}
+
+import scala.collection.immutable.SortedSet
+import java.util.concurrent.locks.ReentrantLock
+import scala.sys.exit
+
 object BddFormula {
 
-  class Bdd
+  val factory: BDDFactory = JFactory.init(1000,100)
```

Comment:
What are these values? and are they well choosen :)

Reply:
They are taken from here: https://buddy.sourceforge.net/manual/main.html (website for BuDDy, which the JavaBDD takes inspiration from). I used the small test examples values, but they should probably be larger

Reply:
Although since our BDDs are generally very small, maybe these values are actually good?